RecoDataProducts/inc/TrackCaloAssns.hh

```diff
@@ -3,7 +3,7 @@
 
 #include "RecoDataProducts/inc/KalRepCollection.hh"
 #include "RecoDataProducts/inc/KalRepPtrCollection.hh"
-#include "RecoDataProducts/inc/CaloClusterCollection.hh"
+#include "RecoDataProducts/inc/CaloCluster.hh"
```

Comment:
this data-product is deprecated. It's used nowhere. Shall we delete it?

Reply:
Sure, but I would make this a separate PR to avoid confusion